src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -19,12 +19,12 @@
 #
 ################################
 
-# Base directory: C:\Projects\sonarlint-visualstudio\src\Integration.Vsix\bin\Debug\VS2022\net472
+# Base directory: D:\repos\sq\slvs\src\Integration.Vsix\bin\Debug\VS2022\net472
 Include patterns:
 - 'Sonar*'
 Exclude patterns:
 - 'SonarLint.Secrets.DotNet*'
-# Number of matches: 19
```

Comment:
I must have got this value wrong before when dealing with the merge conflicts.
It didn't make any difference, though, since the AsmRef tool ignores comments i.e. it wouldn't have complained about the files being different because of this.